### ISSUE TITLE
Neuer Mitarbeiter > Alle Freien Dienstnummern anzeigen fehlerhaft

### ISSUE THREAD

**Issue by danielsteiner** [geraldbaeck/NIUsLittleHelper#50]
Ich bin gestern draufgekommen, dass das Feature "Alle freien Dienstnummern anzeigen" fehlerhaft ist - es wird z.b. der West Schulungs User als frei angezeigt, obwohl der belegt ist.

**Comment by danielsteiner** [comment]
@sebastiankuttnig hast du Zeit dir das anzuschaun? Wenn nicht schau ich dass ich das demnächst mal fixe.

**Comment by sebastiankuttnig** [comment]
Ich kann das Problem nicht reproduzieren, der West-Schulungs-Benutzer (1998) wird bei mir nicht als frei angezeigt...? Auch die anderen angezeigten Dienstnummern sind alle frei.

**Comment by danielsteiner** [comment]
Also mir wird 1026 und 1998 als frei angezeigt - auf beiden NB PCs auf der West und auf meinem privaten Laptop 

![image](https://user-images.githubusercontent.com/1122125/33715572-52f8f0a6-db53-11e7-8978-6a1ed50ff44d.png)

**Comment by sebastiankuttnig** [comment]
Ich kann den Fehler einfach nicht reproduzieren, weder auf meinem Home-PC noch auf meinem Laptop. Eventuell hängt es mit den NIU Berechtigungen zusammen - kannst du mal überprüfen ob du in dem Dropdown unter "Kommando" die fälschlich ausgegebenen Mitarbeiter siehst? Also ist "West Schulung" bei dir als Mitarbeiter unter "Kommando" auswählbar?

**Comment by danielsteiner** [comment]
Nope, sind nicht auswählbar. Anscheinend wurde hier etwas geändert. Unter Liste / Ausdruck find ich sie, under KDO nicht.

**Comment by danielsteiner** [comment]
Da du das ja bisher über die KDO Liste gemacht hast und hier nicht jeder jeden sieht würde ich vorschlagen, dass wir das sobald die Funktion für Liste/Ausdruck fertig ist auch auf das umbauen - dann sollte der "Fehler" auch wieder behoben sein.
